scripts/arabica.sh

```
@@ -22,17 +22,17 @@ echo ""
 
 # Ask the user for confirmation before deleting the existing celestia-app home
 # directory.
-read -p "Are you sure you want to delete: $CELESTIA_APP_HOME? [y/n] " response
+read -r -p "Are you sure you want to delete: $CELESTIA_APP_HOME? [y/n] " response
```

Shellcheck (0.10.0)

[warning] In POSIX sh, read -p is undefined.

(SC3045)